### PR TITLE
tests: fixes for fde and devmode-snap-seeded-dangerous nested tests

### PR DESCRIPTION
**Pull request by sergiocazzolato**
This change:

1. renames models used for fde tests
2.  fix devmode-snap-seeded-dangerous
3.  adapt some nested tests to run in uc24

### CODE REVIEW

File: tests/lib/assertions/classic-model-24.assert [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+grade: dangerous
+serial-authority:
+  - generic
+snaps:
+  -
+    default-channel: 24/edge
+    name: pc
+    type: gadget
+  -
+    default-channel: 24/edge
```
Comment (alfonsosanchezbeato) [inline tests/lib/assertions/classic-model-24.assert:19]: It should be 24/beta to make sure the kernel is signed with Canonical keys - not sure now if this is relevant for the tests where we use the assertion but better be safe.

**Review by alfonsosanchezbeato — COMMENTED** [review]
Just one minor thing

File: tests/lib/assertions/classic-model-24.assert (hunk 1 above)
Comment (sergiocazzolato) [reply tests/lib/assertions/classic-model-24.assert]: done

**Review by alfonsosanchezbeato — APPROVED** [review]
LGTM, thanks

**Review by valentindavid — APPROVED** [review]
That looks good. Thank you.

**Pull request merged**